coords.py

```diff
@@ -75,19 +75,19 @@ def to_sgf(coord):
     return _SGF_COLUMNS[coord[1]] + _SGF_COLUMNS[coord[0]]
 
 
-def from_kgs(kgsc):
-    """Converts from a KGS coordinate to a MiniGo coordinate."""
-    if kgsc == 'pass':
+def from_gtp(gtpc):
+    """Converts from a GTP coordinate to a MiniGo coordinate."""
+    gtpc = gtpc.upper()
+    if gtpc == 'PASS':
         return None
-    kgsc = kgsc.upper()
-    col = _KGS_COLUMNS.index(kgsc[0])
-    row_from_bottom = int(kgsc[1:])
+    col = _GTP_COLUMNS.index(gtpc[0])
+    row_from_bottom = int(gtpc[1:])
     return go.N - row_from_bottom, col
 
 
-def to_kgs(coord):
-    """Converts from a MiniGo coordinate to a KGS coordinate."""
+def to_gtp(coord):
+    """Converts from a MiniGo coordinate to a GTP coordinate."""
     if coord is None:
         return 'pass'
```

Comment:
should this be PASS?

Reply:
The GTP spec uses lowercase for pass in its examples, so lets follow that convention.